rclcpp_action/include/rclcpp_action/client.hpp

```diff
@@ -130,7 +130,7 @@ class ClientBase : public rclcpp::Waitable
     rclcpp::node_interfaces::NodeLoggingInterface::SharedPtr node_logging,
     const std::string & action_name,
     const rosidl_action_type_support_t * type_support,
-    const rcl_action_client_options_t & options);
+    const rcl_action_client_options_t & options = rcl_action_client_get_default_options());
```

Comment:
This change seems unnecessary to me. I think we should not make this optional.

Reply:
agree, thanks for the catch! fixed.